proto/did/did.proto

```diff
@@ -6,28 +6,23 @@ option go_package = "github.com/medibloc/panacea-core/x/did/types";
 import "gogoproto/gogo.proto";
 
 message DIDDocument {
-  repeated string contexts = 1 [json_name = "@contexts"];;
-  string ID = 2 [json_name = "id"];
+  string contexts = 1 [json_name = "@context", (gogoproto.jsontag) = "@context", (gogoproto.customtype) = "Contexts"];
```

Comment:
(gogoproto.jsontag) : Change json name. Not used jsonpb. Used json

Reply:
(gogoproto.customtype) : Use custom go type. Required implements MarshalTo, UnmarshalTo(), Size()